framework/helpers/BaseInflector.php

```diff
@@ -438,12 +471,24 @@ public static function slug($string, $replacement = '-', $lowercase = true)
      * of the helper.
      *
      * @param string $string input string
+     * @param string|\Transliterator $transliterator either a [[Transliterator]] or a string
+     * from which a [[Transliterator]] can be built. When $transliterator is a string, the method will
+     * try to treat is as an alias name and use transliterator value from [[transliteratorAliases]] array.
      * @return string
+     * @see transliteratorAliases
      */
-    protected static function transliterate($string)
+    public static function transliterate($string, $transliterator = null)
```

Comment:
This is potential BC break. Not a serious one but should be mentioned in UPGRADE guide.